release/README.md

```diff
@@ -0,0 +1,18 @@
+
+## Squid Release Automation
+
+### mk-cfgman-docs.sh
+
+Publish the latest [Squid Configuration Guide](http://www.squid-cache.org/Doc/config/).
+
+### mk-langpack.sh
+
+Publish the latest [Squid Translation Package](http://www.squid-cache.org/Versions/langpack/).
+
+### mk-release-series.sh
+
+Branch and publish a new [Squid Series](https://wiki.squid-cache.org/ReleaseSchedule).
```

Comment:
For consistency with other entries, and to highlight the fact that the script automatically increments the current series number by one:

```suggestion
Branch and publish the next [Squid Series](https://wiki.squid-cache.org/ReleaseSchedule).
```